web_disable_export_group/static/src/js/disable_export_group.js

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2016 Onestein
+   Copyright 2018 Tecnativa - David Vidal
+   License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). */
+odoo.define("web_disable_export_group", function(require) {
+"use strict";
+
+    var core = require("web.core");
+    var Sidebar = require("web.Sidebar");
+    var session = require("web.session");
+    var ListController = require("web.ListController");
+    var _t = core._t;
+
+    Sidebar.include({
+        _addItems: function (sectionCode, items) {
+            var _items = items;
+            if (!session.is_superuser && sectionCode === 'other' && items.length && !session.group_export_data) {
+                _items = _.reject(_items, {label:_t("Export")});
+            }
+            this._super(sectionCode, _items);
+        },
+    });
+
+    ListController.include({
+        _updateButtons: function (mode) {
+            if (this.$buttons) {
```

Comment:
Not sure about this total overwrite of the super function, what if we called super and then conditionally hide/show the element? Other people might override here and by not calling super we can break stuff.